pkg/cli/cmds/server.go

```diff
@@ -260,6 +262,12 @@ func NewServerCommand(action func(*cli.Context) error) cli.Command {
 				Usage:       "(components) Disable " + version.Program + " default network policy controller",
 				Destination: &ServerConfig.DisableNPC,
 			},
+			cli.StringFlag{
```

Comment:
Should this be a StringSliceFlag like the --disable list?

Reply:
yes will fix, thanks